Profiler_Test/ProfilerTest.cs

```diff
@@ -13,7 +13,19 @@ namespace Cqse.Teamscale.Profiler.Dotnet
 	/// </summary>
 	[TestFixture]
 	public class ProfilerTest : ProfilerTestBase
-	{
+	{
+		/// <summary>
+		/// Makes sure that processes not matching the given process name are not profiled.
+		/// </summary>
+		[TestCase("w3wp.exe", 0)]
+		[TestCase("ProfilerTestee.exe", 1)]
+		[TestCase("profilerTesTEE.EXE", 1)]
+		public void TestProcessSelection(string process, int expectedTraces)
+		{
+			List<FileInfo> traces = RunProfiler("ProfilerTestee.exe", arguments: "none", lightMode: true, bitness: Bitness.x86, environment: new Dictionary<string, string> { { "COR_PROFILER_PROCESS", process } });
+			Assert.That(traces, Has.Count.EqualTo(expectedTraces), "Got the wrong number of traces.");
+		}

```

Comment:
You can return an int from the function and expect that with Result=1 in the annotation

Reply:
Seems they changed that to ExpectedResult